app/javascript/src/QuickStarts/templates/learnAboutApplicationPlans.yaml

```diff
@@ -0,0 +1,51 @@
+# Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
+metadata:
+  name: learning-about-application-plans
+  # you can add additional metadata here
+  instructional: true
+spec:
+  displayName: Application plans
+  durationMinutes: 15
```

Comment:
I'm not sure about the timing here, what do you think about reducing the duration?